mlir/lib/Dialect/Vector/Transforms/VectorUnroll.cpp

```diff
@@ -631,6 +658,123 @@ struct UnrollGatherPattern : public OpRewritePattern<vector::GatherOp> {
   vector::UnrollVectorOptions options;
 };
 
+// clang-format off
```

Comment:
Let's avoid `clang-format off` and instead try to trim the comment to 80 cols. You have multiple options:
```cpp
// OPTION 1 (split across mulitple lines)
// %result_0 = vector.insert_strided_slice %slice_0, %cst 
//     {offsets = [0, 0], strides = [1]} 
//     : vector<4xf32> into vector<4x4xf32>
//
// OPTION 2 (skip some info, shorten names)
// %s0 = vector.load %base[%indices] : memref<4x4xf32>, vector<4xf32>
// %res_0 = vector.insert_strided_slice %s0, %cst  {offsets = [0, 0], strides = [1]} ...
```